src/mono/wasm/runtime/es6/dotnet.es6.lib.js

```diff
@@ -5,16 +5,16 @@
 "use strict";
 
 #if USE_PTHREADS
-const usePThreads = `true`;
+const usePThreads = true;
 const isPThread = `ENVIRONMENT_IS_PTHREAD`;
 #else
-const usePThreads = `false`;
+const usePThreads = false;
 const isPThread = `false`;
```

Comment:
shouldn't this be de-stringified as well?

Reply:
no, this will be emitted into the final JS code.